karma.conf.js

```diff
@@ -1,72 +1,76 @@
-'use strict';
-
-/* global process */
-
-// configures browsers to run test against
-// any of [ 'ChromeHeadless', 'Chrome', 'Firefox', 'IE', 'PhantomJS' ]
-var browsers =
-  (process.env.TEST_BROWSERS || 'PhantomJS')
-    .replace(/^\s+|\s+$/, '')
-    .split(/\s*,\s*/g)
-    .map(function(browser) {
-      if (browser === 'ChromeHeadless') {
-        process.env.CHROME_BIN = require('puppeteer').executablePath();
-
-        // workaround https://github.com/GoogleChrome/puppeteer/issues/290
-        if (process.platform === 'linux') {
-          return 'ChromeHeadless_Linux';
-        }
-      }
-
-      return browser;
-    });
-
-
-module.exports = function(karma) {
-  karma.set({
-
-    frameworks: [
-      'browserify',
-      'mocha',
-      'sinon-chai'
-    ],
-
-    files: [
-      'test/**/*Spec.js'
-    ],
-
-    preprocessors: {
-      'test/**/*Spec.js': [ 'browserify' ]
-    },
-
-    reporters: [ 'spec' ],
-
-    browsers: browsers,
-
-    customLaunchers: {
-      ChromeHeadless_Linux: {
-        base: 'ChromeHeadless',
-        flags: [
-          '--no-sandbox',
-          '--disable-setuid-sandbox'
-        ],
-        debug: true
-      }
-    },
-
-    autoWatch: false,
-    singleRun: true,
-
-    // browserify configuration
-    browserify: {
-      debug: true,
-      transform: [
-        [ 'babelify', { global: true } ],
-        [ 'stringify', {
-          global: true,
-          extensions: [ '.css' ]
-        } ]
-      ]
-    }
-  });
-};
+'use strict';
+
+/* global process */
+
+// configures browsers to run test against
+// any of [ 'ChromeHeadless', 'Chrome', 'Firefox', 'IE', 'PhantomJS' ]
+var browsers =
+  (process.env.TEST_BROWSERS || 'PhantomJS')
+    .replace(/^\s+|\s+$/, '')
+    .split(/\s*,\s*/g)
+    .map(function(browser) {
+      if (browser === 'ChromeHeadless') {
+        process.env.CHROME_BIN = require('puppeteer').executablePath();
+
+        // workaround https://github.com/GoogleChrome/puppeteer/issues/290
+        if (process.platform === 'linux') {
+          return 'ChromeHeadless_Linux';
+        }
+      }
+
+      return browser;
+    });
+
+var NODE_ENV = process.env.NODE_ENV;
+
+module.exports = function(karma) {
+  karma.set({
+
+    frameworks: [
+      'browserify',
+      'mocha',
+      'sinon-chai'
+    ],
+
+    files: [
+      'test/**/*Spec.js',
+      'dist/diagram-minimap.' + (NODE_ENV === 'production' ? 'production.min' : 'development') + '.js'
+    ],
+
+    preprocessors: {
+      'test/**/*Spec.js': ['browserify']
+    },
+
+    reporters: ['spec'],
+
+    browsers: browsers,
+
+    customLaunchers: {
+      ChromeHeadless_Linux: {
+        base: 'ChromeHeadless',
+        flags: [
+          '--no-sandbox',
+          '--disable-setuid-sandbox'
+        ],
+        debug: true
+      }
+    },
+
+    autoWatch: false,
+    singleRun: true,
+
+    // browserify configuration
+    browserify: {
+      debug: true,
+      transform: [
+        ['babelify', {

```

Comment:
Can you maybe make sure we always have spaces between the squared brackets (as it was before):

```js
['babelify', ...  --> [ 'babelify' , ...
```